.github/workflows/lint-ui.yaml

```diff
@@ -17,10 +17,18 @@ jobs:
       - name: Checkout OSS Teleport
         uses: actions/checkout@v3
 
+      - name: Determine Toolchain Versions
+        run: |
+          echo NODE_VERSION=$(make -s -C build.assets print-node-version) >> $GITHUB_ENV
+
+      - name: Print versions
+        run: |
+          echo "node: ${NODE_VERSION}"
+
       - name: Setup Node Toolchain
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: ${{ env.NODE_VERSION }}
```

Comment:
WDYT of that? We could just change it to node-version: 18, but perhaps it's better to stick to one version everywhere?